perf_test/blas/blas1/KokkosBlas_dot_mv_perf_test.cpp

```diff
@@ -43,7 +43,7 @@
 */
 
 #include <Kokkos_Core.hpp>
-#include <blas/KokkosBlas1_dot.hpp>
+#include <src/KokkosBlas1_dot.hpp>
```

Comment:
Can blas/src not be added to the include path in cmake?

Reply:
Actually that's already the case, I am not sure that the src and/or blas was ever necessary.